sig-scheduling/annual-report-2021.md

```diff
@@ -23,45 +24,46 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
+     - [2249 - Multi-scheduling Profiles](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/1451-multi-scheduling-profiles) - 1.22
    - Beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.beta
+     - [2249 - Namespace Selector for Pod Affinity](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/2249-pod-affinity-namespace-selector) - 1.22
+     - [1923 - Prefer Nominated Node](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/1923-prefer-nominated-node) - 1.22   
+     - [2458 - Resource Fit Scoring Strategy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/2458-node-resource-score-strategy) - 1.22
+     - [2891 - Simplified Scheduler Config](https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/2891-simplified-config/kep.yaml) - 1.22
+     - [785 - Scheduler Component Config API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/785-scheduler-component-config-api) - 1.23
+     - [2926 - Job Mutable Scheduling Directives](https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/2926-job-mutable-scheduling-directives) - 1.23
    - Alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md) - $milestone.alpha
+     - None
    - Pre-alpha
-     - [$kep-number - $title](https://git.k8s.io/community/$link/README.md)
+     - None
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - https://github.com/kubernetes-sigs/kube-scheduler-simulator is short of reviewers/owners.
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - Diversity
+   - Number of contributors
+   - Meetings attendance
+   
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - It looks up-to-date 
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - Nothing special.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
```

Comment:
I can provide some viewpoints from my side, I'm working at a paas company, we build our product ecosystem on kubernetes, the knowledge of kubernetes is quite important to us. So I think there are two main reasons with us:
1. Following up the newest features in kubernetes at source code level.
2. Contributing our features to the community and leverage the power of community to optimize them.

Some poor thoughts.

Reply:
> Following up the newest features in kubernetes at source code level.

This is part of a larger process that we have in the k8s release process, I am not sure this is sig-scheduling specific. But scheduler's new features are being documented in the k8s public doc as well as release notes. Are you saying that  something is still lacking? if so what is it?

> Contributing our features to the community and leverage the power of community to optimize them.

We have the KEP process for that, do you find something lacking in the KEP process?

Reply:
Aha, I was just explaining why we'd like to contribute to kube community, not specific to sig/scheduling. And it's a great experience about KEP for now, we also draw on this process to help designing our architecture in the earlier time.

Reply:
sg, I guess we can resolve this then.